11/homework_11.py

```diff
@@ -0,0 +1,62 @@
+# ==================== 11.1
+# Написати декоратор
+# імпортуємо datetime
+from datetime import datetime
+
+# створимо декоратор, який буде доповнювати функції їх назвою та часом виклику
+def deco(func):
+    def inner(*args, **kwargs):
+        # виведем назву функції, за допомогою методу __name__
+        print(f"\nFunction name is: '{func.__name__}'")
+        # виведем час виклику функції
+        now = datetime.now()
+        print(f"Today's date is: {now.strftime('%d.%m.%Y %H:%M')}")
+        # викликаємо функцію, яка буде декоруватись
+        func(*args, **kwargs)
```

Comment:
Use `return` statement here to be able to decorate functions that return some value